CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Renamed `api.(yaml|json)` to `scanapi.yaml` [#222](https://github.com/scanapi/scanapi/issues/20://github.com/scanapi/scanapi/pull/222)
 - Remove top-level `api` key in `scanapi.yaml`. [#231](https://github.com/scanapi/scanapi/pull/231)
 - Renamed `project-name`, `hide-request` and `hide-response` to use underscore. [#228](https://github.com/scanapi/scanapi/issues/228)
+- Changed command `scanapi <spec_file>` to `scanapi run <spec_file>`. [#247](https://github.com/scanapi/scanapi/pull/247) 
```

Comment:
@abreumatheus just a wild guess here can you try 

```
- Renamed change command "scanapi <spec_file>" to "scanapi run <spec_file>" [#247](https://github.com/scanapi/scanapi/pull/247)
```

Reply:
It's weird that the stack trace of `circleci` dosen't give out any error message.

Reply:
Weird 🤔. Diff outputs the same values.

Reply:
@Pradhvan @barbosa Changed it, still failing :/

Reply:
@abreumatheus Now you got a different error. See:

```diff
-- Changed command "scanapi <spec_file>" to "scanapi run <spec_file>" [#247](https://github.com/scanapi/scanapi/pull/247) 
+- Changed command "scanapi " to "scanapi run " [#247](https://github.com/scanapi/scanapi/pull/247)
```

Apparently the tool is cutting out everything in between `< >`

Reply:
> @abreumatheus Now you got a different error. See:
> 
> ```diff
> -- Changed command "scanapi <spec_file>" to "scanapi run <spec_file>" [#247](https://github.com/scanapi/scanapi/pull/247) 
> +- Changed command "scanapi " to "scanapi run " [#247](https://github.com/scanapi/scanapi/pull/247)
> ```
> 
> Apparently the tool is cutting out everything in between `< >`

Nicely spotted! Will fix it :)

Reply:
@barbosa Yeah, it wasn't it. Apparently Circle CI just hates me 🤣

Reply:
@abreumatheus was it just the empty space in the end that was causing this issue ?

Reply:
> @abreumatheus was it just the empty space in the end that was causing this issue ? 

Honestly, i dont even know, just ran the linter locally and saved the output to a new file.

Reply:
okay. I saw the diff in the end and the only thing I could make out of it was that space xp

Reply:
wow, this was a nice team effort, what a battle 😂

👏👏👏